<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skplayerClass</class>
 <widget class="QWidget" name="skplayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>skplayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="skplayer.qrc">
    <normaloff>:/skplayer/Resources/play_128px_1194928_easyicon.net.ico</normaloff>:/skplayer/Resources/play_128px_1194928_easyicon.net.ico</iconset>
  </property>
  <widget class="VideoWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/skplayer/Resources/Folder_open_246px_1198271_easyicon.net.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/skplayer/Resources/play_703px_1194928_easyicon.net.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playTime1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(242, 255, 228);
font: 9pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>000:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTime1_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(242, 255, 228);
font: 9pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>000:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="sp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 255, 242);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="sSlider" name="playslider">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="ModelEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>670</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLb">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>790</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>score&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MeanEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>730</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ModelButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>670</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Model文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnalysisBt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>790</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Analysis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScoreIp">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>790</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProtoEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>700</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OutputDir">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>760</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>存放位置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InitCaffe">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>790</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>InitCaffe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProtoButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>700</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Proto文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutputDirEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>760</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MeanButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>730</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean文件</string>
   </property>
  </widget>
  <widget class="QTableView" name="Seginfo">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>321</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="LogText">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>600</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>sSlider</class>
   <extends>QSlider</extends>
   <header>sslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="skplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>skplayerClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>907</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>skplayerClass</receiver>
   <slot>sliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>1109</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>skplayerClass</receiver>
   <slot>sliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>994</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>skplayerClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPressed()</slot>
  <slot>sliderReleased()</slot>
  <slot>play()</slot>
 </slots>
</ui>
